modules/reference/pages/tunable-properties.adoc

```diff
@@ -1870,9 +1870,9 @@ The lower bound for the topic property `segment.ms` and the cluster property `lo
 
 *Units*: milliseconds
 
-*Default*: 60000 (60 sec)
+*Default*: 600000 (10 minutes)
 
-*Range*: [60000 (60 sec), ...]
+*Range*: [600000 (10 minutes), ...]
```

Comment:
The properties `log_segment_ms_min` and `log_segment_ms_max` don't really have a range in the sense that these two properties define the min and max values of the valid range of `log_segment_ms`. Technically they have a range that is more like `[0, ...)`.